api/src/controllers/team.js

```diff
@@ -92,21 +92,23 @@ function getTeam(req, res, next) {
  * @param {*} res response object - updated team object
  * @returns 201: returns updated team details
  */
-async function addTeam(req, res, next) {
+function addTeam(req, res, next) {
```

Comment:
This controller name doesn't seem too appropriate IMO. Add team implies a list of teams created in a particular session. This should be renamed to `createTeam`? @vgoh1 thoughts?